dictsource/be_list

```diff
@@ -0,0 +1,81 @@
+// This file is UTF8 encoded
+// Spelling-to-phoneme words for Belarusian
+
+// Letter names
+а       a
+_б      bE
+б       bE  $atend
+_в      vE
+в       vE  $atend
+г       QE
+д       dE
+е       jE
+ё       j;O
+_ж      z.E
+ж       z.E  $atend
+з       zE
+і       i
+й       i n;Esklad;OvajE
+к       ka
+л       l
+м       m
+н       En
+о       ;O
+п       pE
+р       Er
+_с      Es
+с       Es  $atend
+т       tE
+у       u
+ў       u n;Esklad;OvajE
+ф       Ef
+х       xa
+ц       tsE
+ч       tS;E
+ш       s.a
+ы       y
+ь       m;ak;:i znak
+ю       ju
+я       ja
+'       ap;Ostraf
```

Comment:
Is semicolon here the sign of softness? If yes then why `p` is soft here?

Reply:
thanks, it was copy-pasted from:

https://star-wiki.ru/wiki/Belarusian_alphabet

Reply:
removed